server/models/pattern/core/pattern.go

```diff
@@ -532,12 +534,32 @@ func createPatternServiceFromK8s(manifest map[string]interface{}) (string, Servi
 	if apiVersion == "" || kind == "" {
 		return "", Service{}, ErrCreatePatternService(fmt.Errorf("empty apiVersion or kind in manifest"))
 	}
-	w := GetWorkloadsByK8sAPIVersionKind(apiVersion, kind)
 
-	if len(w) == 0 {
+	// Get MeshModel entity with the selectors
+	res := regManager.GetEntities(&meshmodelv1alpha1.ComponentFilter{
+		Name:       kind,
+		ModelName:  "kubernetes", // currently hardcoded for kubernetes manifest conversion
```

Comment:
Please omit `modelname` and greedy is by default false.

Reply:
Done